gears/src/baseapp/abci.rs

```diff
@@ -206,7 +206,7 @@ impl<DB: Database, PSK: ParamsSubspaceKey, H: ABCIHandler, AI: ApplicationInfo>
     fn commit(&self) -> ResponseCommit {
         info!("Got commit request");
 
-        let height = self.block_height();
+        let height = self.get_block_header().unwrap().height;
```

Comment:
unwrap?

Reply:
It's effectively a TODO. It would be good to find a way of making the header not be an `Option`. One way might be to initialize it to an "empty" header.